src/test/java/com/epam/tc/hw5/ex1/steps/AbstractStep.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.tc.hw5.ex1.steps;
+
+import com.epam.tc.hw5.ex1.cucumber.context.TestContext;
+import com.epam.tc.hw5.pages.DifferentPage;
+import com.epam.tc.hw5.pages.GdiGitHubSite;
+import com.epam.tc.hw5.pages.components.LeftSideMenuComponents;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.WebDriver;
+
+public abstract class AbstractStep {
```

Comment:
you don't have to create separated AbstractStep classes for each exercise. fix it.